src/components/molecules/LabelsSelect/LabelsSelect.tsx

```diff
@@ -17,7 +17,7 @@ import {composeLabels} from './utils';
 
 type LabelsSelectProps = {
   onChange?: (value: readonly Option[]) => void;
-  defaultLabels?: readonly Option[];
+  defaultLabels?: Record<string, string> | Option[];
```

Comment:
Based on implementation - shouldn't it be `Record<string, Option>` instead of `Record<string, string> | Option[]`?